src/plugin/settings/engine.cpp

```diff
@@ -267,21 +272,25 @@ Result<const ISettingsResponse *> Engine::settingsReceived()
         {
 #if defined(__linux__)
             // On Linux we need to create the folder if it does not yet exist
+            NX_PRINT << "creating the folder if it does not yet exist" << std::endl;
```

Comment:
Since this is in the else case, we could use more certain language here, and say that the mount directory does not exist.